packages/wonder-blocks-data/src/components/data.js

```diff
@@ -5,15 +5,15 @@ import {Server} from "@khanacademy/wonder-blocks-core";
 import {RequestFulfillment} from "../util/request-fulfillment.js";
 import InterceptContext from "./intercept-context.js";
 import {useServerEffect} from "../hooks/use-server-effect.js";
-import {resultFromCacheEntry} from "../util/result-from-cache-entry.js";
+import {resultFromCachedResponse} from "../util/result-from-cache-entry.js";
```

Comment:
I like this.

Reply:
The good fortune of using find/replace 😁

Reply:
I'll rename the file to match